<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="aj-305E4A-305E5A-305E7A" ignored="0" name="Test" tests="1" failures="0" timestamp="22 May 2017 05:45:49 GMT" time="5.822" errors="0">
  <testcase name="@BeforeClass beforeTest" time="5.706" classname="res.one">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at res.one.beforeTest(one.java:40)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass beforeTest -->
  <testcase name="login" time="0.0" classname="res.one">
    <skipped/>
  </testcase> <!-- login -->
</testsuite> <!-- Test -->
